### ISSUE TITLE
Release training loss curves/data for 7B and 1.5B runs

### ISSUE THREAD

**Issue by RobertMcCarthy97** [open-thoughts/open-thoughts#43]
Hi, thanks for open-sourcing this dataset, finding it very useful!

I'm wondering if you could release the training loss data from your final training runs with the 1.5B and 7B models?

I am recreate your training on both model sizes with slightly weaker hyperparameters (namely a lower global batch size, due to having access to less GPUs). It would be useful to know what my loss curves should look like, to help me know if my hyperparameters are performing well.

Any extra info along these lines would be very useful - e.g., what training loss does the model converge to, etc.

Thanks!
